bitacora/patterns/footer.php

```diff
@@ -61,22 +61,8 @@
 	</div>
 	<!-- /wp:columns -->
 
-	<!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+	<!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
```

Comment:
I think we can get rid of this group and just display a paragraph which is aligned to the center

With block group:
<img width="1257" alt="Screenshot 2024-08-01 at 11 46 21" src="https://github.com/user-attachments/assets/91990148-9281-4442-b3c0-f933e20930a4">

Without block group (paragraph only aligned to the center):
<img width="1268" alt="Screenshot 2024-08-01 at 11 46 49" src="https://github.com/user-attachments/assets/27072abd-2e25-469c-8d22-0cd5e74380f9">